docs/_posts/2021-02-15-42.2.19-release.md

```diff
@@ -0,0 +1,32 @@
+---
+title: PostgreSQL JDBC Driver 42.2.19 Released
+date: 2021-02-15 19:19:40 +0000
+categories:
+  - new_release
+version: 42.2.19
+---
+**Notable changes**
+
+
+**Notable Changes**
+- Now the driver use SASLprep normalization for SCRAM authentication fixing some issues with spaces in passwords.
+
+### Changed
+
+### Added 
+- Verify code via forbidden-apis (jdk-internal and jdk-non-portable signatures) [PR #2012](https://github.com/pgjdbc/pgjdbc/pull/2012)
+
+### Fixed
+- Fix "Required class information missing" when old org.jboss:jandex parses pgjdbc classes [issue 2008][https://github.com/pgjdbc/pgjdbc/issues/2008]
+- Fix PGCopyInputStream returning the last row twice when reading with CopyOut API [issue 2016][https://github.com/pgjdbc/pgjdbc/issues/2016]
+- Fix Connnection.isValid() to not wait longer than existing network timeout [PR #2040](https://github.com/pgjdbc/pgjdbc/pull/2040)
+- Use SASLprep normalization for SCRAM authentication [PR #2052](https://github.com/pgjdbc/pgjdbc/pull/2052)
+- Fix DatabaseMetaData.getTablePrivileges() to include views, materialized views, and foreign tables [PR #2049](https://github.com/pgjdbc/pgjdbc/pull/2049)
+
+
+<!--more-->
+
+**Commits by author**
+
+Can't locate WWW/Curl/Easy.pm in @INC (you may need to install the WWW::Curl::Easy module) (@INC contains: /etc/perl /usr/local/lib/x86_64-linux-gnu/perl/5.26.1 /usr/local/share/perl/5.26.1 /usr/lib/x86_64-linux-gnu/perl5/5.26 /usr/share/perl5 /usr/lib/x86_64-linux-gnu/perl/5.26 /usr/share/perl/5.26 /usr/local/lib/site_perl /usr/lib/x86_64-linux-gnu/perl-base) at release_notes_filter.pl line 6.
```

Comment:
@davecramer Missing dep building things?

Reply:
ugh, someone needs to rewrite that thing in groovy